utils/swift_build_support/swift_build_support/products/swift_testing.py

```diff
@@ -0,0 +1,67 @@
+# swift_build_support/products/swift_testing.py -----------------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2024 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+
+import os
+
+from . import cmake_product
+from . import swift
+from . import swiftsyntax
+
+class SwiftTesting(cmake_product.CMakeProduct):
+    @classmethod
+    def is_build_script_impl_product(cls):
+        return False
+
+    @classmethod
+    def is_before_build_script_impl_product(cls):
+        return False
+
+    @classmethod
+    def product_source_name(cls):
+        return "swift-testing"
+
+    @classmethod
+    def get_dependencies(cls):
+        return [swift.Swift,
+                swiftsyntax.SwiftSyntax]
```

Comment:
There is no need to wait for SwiftSyntax -- in the build we are using the swift syntax libraries that we build in the Swift compiler for plugins

```suggestion
        return [swift.Swift]
```